docs/zkapps/o1js/interact-with-mina.md

```diff
@@ -353,43 +406,53 @@ await tx.prove();
 tx.sign([senderPrivateKey]); // senderKey is a PrivateKey
 ```
 
-The example first shows us how we can derive the sender's public key `sender` from its private key `senderPrivateKey`.
+- The example shows how to derive the sender's public key, `sender`, from its private key, `senderPrivateKey`.
+- `.sign()` goes through the transaction and adds signatures on all account updates that:
+  
+    - Need a signature
+    - Whose public key matches one of the private keys that were provided 
+  
+  `.sign()` takes an array, so you could provide multiple private keys for signing. 
 
-Note that `.sign()` takes an array, so you could provide multiple private keys for signing. `.sign()` will go through the transaction and add signatures on all account updates which 1) need a signature and 2) whose public key matches one of the private keys that were provided. In the example above, two account updates are signed with `tx.sign()`: The fee payer and the depositor account update. Both have the `sender` public key on them, which matches `senderPrivateKey.toPublicKey()`.
+In this example, two account updates are signed with `tx.sign()`: The fee payer and the depositor account update. Both have the `sender` public key on them that matches `senderPrivateKey.toPublicKey()`.
 
-:::note
+:::tip
+o1js allows you to load and store private and public keys in Base58 format. 
+:::
```

Comment:
I think the code below should be part of this tip! Otherwise it is ripped from context

Reply:
hmmm... "ripped from content" we don't want that.

I am persuaded